### ISSUE TITLE
Flatpak engine can't access Steam Flatpak files

### ISSUE THREAD

**Issue by a1batross** [FWGS/xash3d-fwgs#1240]
```
Error: W_Open: couldn't open /redacted/.var/app/com.valvesoftware.Steam/.local/share/Steam/steamapps/common/Half-Life/valve/cached.wad: No such file or directory
```

I fixed this for SteamOS 3, but right now I don't remember how. %)

**Comment by a1batross** [comment]
The issue is caused by RoDir that engine can't access absolute paths because of this particular hack:
```
	// some stupid mappers used leading '/' or '\' in path to models or sounds
	if( filepath[0] == '/' || filepath[0] == '\\' )
		filepath++;

	if( filepath[0] == '/' || filepath[0] == '\\' )
		filepath++;
```

**Comment by a1batross** [comment]
I'm not sure if I can just remove this hack. =/

**Comment by a1batross** [comment]
Must be fixed with 1e8c26a527f8f8397d0ba3a665787bd7b9c3e5d6
